packer/scylla_install_image

```diff
@@ -147,6 +147,8 @@ if __name__ == '__main__':
 
     run('systemctl daemon-reload')
     run('systemctl enable scylla-image-setup.service')
+    if distro == 'ubuntu':
```

Comment:
I want to avoid `if distro` here, if it possible.
We may not use CentOS AMI on newer releases but this change still can work on CentOS correct?
If it possible, please drop `if distro` line and also add files to rpm too.

Reply:
Currently it's not gonna work since I didn't add the service to the RPM version,
I'll try adding it and remove this condition.